caravel/assets/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "caravel",
-  "version": "0.1.0",
+  "version": "0.10.1",
```

Comment:
I agree, let's stick with `0.10.0` for this PR, we may want to go "0.10.0.dev1" between releases as I started doing for Airflow, but let's do `0.10.0` here for now.